favicon.go

```diff
@@ -0,0 +1,26 @@
5lP7/+bUu//4cn6//H///892/3/CNH8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wzS/P+e7f7//////f///2EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD///+F/////9Gw+P+aUe//mU7v/5lP7/+ZT+//mU/v/5lP7/+YTe//z6z3//7///9g4v3/B9H8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/CNH8/zfa/f/l+v//////3P///yIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD///8m////1fr3/v/Gnvb/nFTv/5dM7/+YTe//mE7v/5hN7/+WSu//w5j2//////+N6v7/CNH8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8K0vz/ENP8/6Dt/v//////////hwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA////Tf///+X8+f7/3MT6/7eD9P+nZ/H/pGLx/6xw8v/GnPb/7+P8///////f+f//Pdv9/wfR/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8J0vz/a+P9//b9///////U////IwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA////Af///0T////F////+/z6///07P3/8ej9//jy/v/+/v/9////zf///5z////1y/X+/zXa/f8H0fz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/CtL8/wrS/P9c4f3/6fv///////H///9XAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD///8V////XP///53///+8////vv///6P///9l////G////wj///+K////+9L3/v9N3v3/C9L8/wjS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wvS/P8L0vz/C9L8/wrS/P8I0fz/FNT8/3Pl/f/r+///////9////3j///8EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAP///wL///8I////Cf///wIAAAAAAAAAAAAAAAD///8M////h/////Xs+///j+r+/zHZ/P8M0vz/B9H8/wjR/P8J0vz/CdL8/wnS/P8H0fz/B9H8/xLT/P9G3f3/r/D+//j+///////w////dv///wcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA////B////2H////a/v///+P6//+j7v7/ZeL9/z7b/f8s2Pz/KNf8/zDZ/P9H3f3/dOX9/7bx/v/w/P///////v///9H///9T////BAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD///8m////if///9r////7/f////H8///l+v//4vn//+j7///1/f///v////////n////U////ff///x4AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA////Av///yT///9p////qv///9b////s////8v///+r////T////pP///2H///8f////AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD//wH///8AAP/8AD///wAA//AAH///AAD/4AAP//8AAP/AAAOH/wAA/4AAAAH/AAD/AAAAAP8AAP8AAAAAfwAA/gAAAAA/AAD+AAAAAD8AAPwAAAAAPwAA/AAAAAA/AAD8AAAAAD8AAPwAAAAAPwAA/AAAAAAfAAD8AAAAAA8AAPwAAAAAAwAA8AAAAAADAADgAAAAAAEAAMAAAAAAAQAAgAAAAAAAAACAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAEAAIAAAAAAAwAAgAAAAAAHAADAAAAAAA8AAMAAAAAAPwAA8AAAAAA/AAD4AAAAAD8AAPwAAAAAPwAA/AAAAAA/AAD8AAAAAD8AAPwAAAAAPwAA/AAAAAB/AAD8AAAAAH8AAP4AAAAAfwAA/wAAAAD/AAD/gAAAAf8AAP/hwAAD/wAA///gAAf/AAD///gAD/8AAP///AA//wAA////gP//AAA="
```

Comment:
I'm good with hardcoding this for now as it is unlikely to change but it would be nice to have the go file generated by `go generate` to be able to update it.

Reply:
Thanks for the hint. I agree that it can be adjusted later.